src/ensembl/src/shared/components/layout/StandardAppLayout.scss

```diff
@@ -146,6 +146,6 @@ $drawerWindowWidth: 45px;
 
 .drawerClose {
   position: absolute;
-  right: calc(#{$drawerWindowWidth} + 10px);
-  top: 10px;
+  right: calc(#{$drawerWindowWidth} + 30px);
```

Comment:
Are you also seeing the close button right over the scrollbar now?

<img src="https://user-images.githubusercontent.com/6834224/134322414-92930593-a389-4c52-95c4-60c34e62e252.png" width="70" />